src/blocks/BoardListBlock.tsx

```diff
@@ -72,17 +292,28 @@ export interface EmptyProps {
 }
 export interface ItemProps {
   slug: string;
-  description?: string;
+  color: string;
+  avatar: string;
+  link?: LinkWithAction;
+  description: string;
   _onSelected?: (slug: string) => void;
   _selected?: boolean;
   options: { name: string; link: LinkWithAction }[];
+  muted?: boolean;
+  updates?: number | boolean;
+  outdated?: boolean;
+  _onLinkedClicked?: () => void;
```

Comment:
onLinked? Maybe `onLinkClicked`? Feel free to add a comment explaining what this means

Reply:
Also, add a TODO to remove these from the externally-visible types.